cmake/CMakeLists.txt

```
@@ -33,6 +46,21 @@ set_variable_string(HAVE_STORM_GSPN_BOOL ${HAVE_STORM_GSPN})
 set_variable_string(HAVE_STORM_PARS_BOOL ${HAVE_STORM_PARS})
 set_variable_string(HAVE_STORM_POMDP_BOOL ${HAVE_STORM_POMDP})
 
+# Set configuration
+set_variable_string(CARL_DIR ${carl_DIR})
+set_variable_string(CARL_VERSION ${carl_VERSION})
```

```suggestion
set(CARL_DIR ${carl_DIR})
set(CARL_VERSION ${carl_VERSION})
```
`set_variable_string` sets the variable to either `TRUE` or `FALSE`